packages/tslint-config/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@loopback/tslint-config",
+  "version": "1.0.0-1",
+  "description": "",
+  "engines": {
+    "node": ">=8.9"
+  },
+  "author": "IBM",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "peerDependencies": {
+    "tslint": ">=5.11.0"
```

Comment:
Hmm. The idea behind this peer dependency is to specify the minimum version of `tslint` that can understand our rules. Unfortunately, a typical LB4 project does not install `tslint` directly but gets it from `@loopback/build` instead.

As a result, `npm install` can print the following warning:

```
npm WARN @loopback/tslint-config@1.0.0-1 requires a peer of tslint@>=5.11.0 but none is installed. You must install peer dependencies yourself.
```

I am not sure what to do about it. We can remove this peer dependency, but that way we loose control over `tslint` version used with our config. Or we can modify our project templates and example projects to install `tslint` directly, kind of duplicating `@loopback/build`'s dependency on tslint. 

I think in the long term, it may be best to get rid of `lb-tslint` and use https://github.com/Microsoft/tslint-microsoft-contrib to invoke tslint as part of `tsc` build.

@strongloop/loopback-maintainers thoughts?

Reply:
What's worse, some 3rd-party tslint rules like tslint-consistent-codestyle (see https://github.com/strongloop/loopback-next/pull/2156) are loading `tslint` directly and fail if it's not installed as a peer dependency (e.g. because it's sourced from` @loopback/build`). I'll have to deal with this problem in #2156.